src/decider/agent/DeciderAgent.php

```diff
@@ -431,15 +431,15 @@ private function autodecideLicenseType(ItemTreeBounds $itemTreeBounds,
                                          $matches)
   {
     $canDecide = $this->noLicenseConflict($itemTreeBounds, $matches);
-
+    $autoConclude = true;
     if ($canDecide) {
       $canDecide &= $this->allLicenseInType($matches);
     }
 
     if ($canDecide) {
       $this->clearingDecisionProcessor
           ->makeDecisionFromLastEvents($itemTreeBounds, $this->userId,
-              $this->groupId, DecisionTypes::IDENTIFIED, DecisionScopes::ITEM);
+              $this->groupId, DecisionTypes::IDENTIFIED, DecisionScopes::ITEM, $autoConclude);
```

Comment:
Can be simplified as simply passing `true` to `makeDecisionFromLastEvents()` as the variable is not used anywhere else.